tests/unit/shared/test_vic_time.py

```diff
@@ -135,68 +134,64 @@ def test_date2num(feb3_noon, dmy_feb_3_noon):
 
 
 @pytest.mark.skipif(nctime_unavailable, reason=nc_reason)
-def test_dmy_julian_day():
+def test_dmy_julian_day(feb3_noon):
```

Comment:
I don't think that the edits in the file `test_vic_time.py` should be part of this PR. Please move to separate PR.